test/docker-compose.yaml

```diff
@@ -238,12 +244,24 @@ services:
       - "9090"
       - "2112"
     command: [ "./arc", "-api=true", "-config=." ]
+    environment:
+      - ARC_TRACING_DIALADDR
```

Comment:
How does this env var work? There is no value
Don't you have to add the setting in `/test/config/config.yaml`?

Reply:
This env var is to able the usage of env when running docker compose in Makefile:
`ARC_TRACING_DIALADDR=http://arc-jaeger:4317 docker compose -f test/docker-compose.yaml up --build ...
`

If we add it to `/test/config/config.yaml` then we will always have tracing when running the tests locally. With this solution we have 2 separate commands in the Makefile
`run_e2e_tests`
and
`run_e2e_tests_with_tracing`

Reply:
Ah, understand. If it is in config then tracing will be always enabled instead of optional